docs/A-Introduction/compilers.md

```diff
@@ -1,5 +1,9 @@
 ---
+id: compilers
+title: Compilers
 sidebar_position: 3
+slug: /A-Introduction/compilers
```

Comment:
Make sure your slug looks a bit more standardized, based on [this example](https://github.com/Seneca-ICTOER/IPC144/issues/64).

Notice that the letter is removed ("D"), and everything is in lower case:

this `slug: /D-Modularity/pointers` becomes this `slug: /modularity/pointers` 

so in  your case, it would look something like that: 

this `slug: /A-Introduction/compilers` will become this `slug: /introduction/compilers`

Reply:
Thank you, I will fix it :)